Prox/Prox/PlaceDetails/PlaceDetailsCardViewController.swift

```diff
@@ -295,6 +295,12 @@ extension PlaceDetailsCardViewController: UIScrollViewDelegate {
 
     fileprivate func didChangePage(scrollView: UIScrollView) {
         let pageSize = imageCarouselCollectionView.bounds.size
+
+        // Unable to change pages if we don't have anything to page!
```

Comment:
nit: This should be a little clearer that "we don't have anything to page" when the image carousel is empty.